packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.js

```diff
@@ -27,16 +57,19 @@ class Checkbox extends React.Component {
   };
 
   render() {
-    const { className, onChange, isValid, isDisabled, ...props } = this.props;
+    const { className, onChange, isValid, isDisabled, label, ...props } = this.props;
     return (
-      <input
-        {...props}
-        className={css(styles.check, className)}
-        type="checkbox"
-        onChange={this.handleChange}
-        aria-invalid={!isValid}
-        disabled={isDisabled}
-      />
+      <div className={css(styles.check, className)}>
+        <input
+          {...props}
+          className={css(styles.checkInput)}
```

Comment:
The className from prop is not getting passed through

Reply:
That's my question from description of this PR. Do we want to pass className to the input or to wrapperDiv? Or should I create classNames props separately for input, label, wrapperdiv?

Reply:
maybe the wrapper div, if user needs to target the input for styling they will be able to, but can also apply style to the wrapper

Reply:
Yep that's good thanks